care/facility/api/serializers/prescription.py

```diff
@@ -57,6 +57,20 @@ def validate(self, attrs):
                 MedibaseMedicine, external_id=attrs["medicine"]
             )
 
+        if not self.instance:
+            if Prescription.objects.filter(
+                consultation__external_id=self.context["request"].parser_context[
+                    "kwargs"
+                ]["consultation_external_id"],
+                medicine=attrs["medicine"],
+                discontinued=False,
+            ).exists():
+                raise serializers.ValidationError(
+                    {
+                        "medicine": "This medicine is already prescribed to this patient. Please discontinue the existing prescription to prescribe again."
```

Comment:
```suggestion
                        "medicine": (
                            "This medicine is already prescribed to this patient. "
                            "Please discontinue the existing prescription to prescribe again."
                            )
```

Reply:
Copilot had suggested me this 😅 I ignored it